cns/client_test.go

```diff
@@ -1251,14 +1260,98 @@ func TestClient(t *testing.T) {
 		}
 		t.Logf("volume:%q deleted sucessfully", volumeID)
 	}
+
+	// Test CnsReconfigVolumePolicy API
+	if spbmPolicyId4Reconfig != "" {
+		var cnsVolumeCreateWithProfileSpecList []cnstypes.CnsVolumeCreateSpec
+		cnsVolumeCreateSpecWithProfile := cnstypes.CnsVolumeCreateSpec{
+			Name:       "pvc-901e87eb-c2bd-11e9-806f-005056a0c9a0-1",
+			VolumeType: string(cnstypes.CnsVolumeTypeBlock),
+			Datastores: dsList,
+			Metadata: cnstypes.CnsVolumeMetadata{
+				ContainerCluster: containerCluster,
+			},
+			BackingObjectDetails: &cnstypes.CnsBlockBackingDetails{
+				CnsBackingObjectDetails: cnstypes.CnsBackingObjectDetails{
+					CapacityInMb: 5120,
+				},
+			},
+		}
+		cnsVolumeCreateWithProfileSpecList = append(cnsVolumeCreateWithProfileSpecList, cnsVolumeCreateSpecWithProfile)
+		t.Logf("Creating volume using the spec: %+v", pretty.Sprint(cnsVolumeCreateWithProfileSpecList))
```

Comment:
Instead of creating a new volume, maybe you should use the existing volume created in this test, and apply the policy set in the env variable.

Reply:
I think volume created in the test get deleted in line https://github.com/vmware/govmomi/blob/d8e3357f206f310cba05748e23587a094f22ee71/cns/client_test.go#L915 after that for each API new volumes are created and deleted, so I was following the same